Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -305,3 +286,59 @@ you can export the settings from this cluster with the following steps.
 
 !!! important
     For other clusters to connect to storage in this cluster, Rook must be configured with a networking configuration that is accessible from other clusters. Most commonly this is done by enabling host networking in the CephCluster CR so the Ceph daemons will be addressable by their host IPs.
+
+## Upgrades
+
+1. If consumer cluster doesn't have restricted caps, this will upgrade all the default csi-users (non-restricted)
```

Comment:
Generally, numbered items are taken to be steps that users perform in order. I suspect some users might try to do both without considering that these are intended to be alternatives. 

I would suggest writing a preface statement just under `## Upgrades` that explains there are two scenarios and users should follow the one scenario for their use-case. We can still use numbers, but we should at least explain how they are used to help readers